sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/v2ResolutionPlans.scala

```diff
@@ -188,14 +188,15 @@ case class ResolvedFieldPosition(position: ColumnPosition) extends FieldPosition
 case class ResolvedPersistentView(
     catalog: CatalogPlugin,
     identifier: Identifier,
-    viewSchema: StructType) extends LeafNodeWithoutStats {
+    viewSchema: StructType,
```

Comment:
I'm wondering if we should put the entire view metadata. It can be `viewMetadata: CatalogTable`. We can refactor it when we have v2 views.

Reply:
Good point!